riff-raff.yaml

```diff
@@ -9,7 +9,8 @@ templates:
   frontend:
     type: autoscaling
     parameters:
-      bucket: aws-frontend-artifacts
+      bucketSsmKey: /account/services/dotcom-artifact.bucket
+      bucketSsmLookup: true
```

Comment:
```suggestion
     
```

It turns out we only need `bucketSsmKey`. See [here](https://github.com/guardian/dotcom-rendering/pull/6560#discussion_r1030581193)